ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/test/test_systemd.ml

```diff
@@ -0,0 +1,47 @@
+let _ =
+  let module Daemon = Xapi_stdext_unix.Unixext.Daemon in
+  match Array.to_list Sys.argv with
```

Comment:
Would it have been convenient to use the `Arg` module from the standard library? We often use `Cmdliner` but it requires quite a bit of boilerplate code that is probably not worth it here.

Reply:
Sure! I've followed `ocaml/forkexecd/test/fe_test.ml` here for simplicity, but already forgot a doc string for one of the options. Switched to `Arg` instead.